packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/About.1.doc.mdx

```diff
@@ -1,23 +1,105 @@
 
-# Terra CompactInteractiveList
+# Terra Compact Interactive List
 
 <Notice variant="caution" ariaLevel="2">
 Terra Compact Interactive List is currently under alpha release. There may be breaking changes between releases until it is stable with the 1.0 release.
 </Notice>
 
+## Table of Contents
+* [Overview](/components/cerner-terra-framework-docs/compact-interactive-list/about#overview)
+* [Design Decisions](/components/cerner-terra-framework-docs/compact-interactive-list/about#design-decisions)
+* [Examples](/components/cerner-terra-framework-docs/compact-interactive-list/about#examples)
+* [Accessibility](/components/cerner-terra-framework-docs/compact-interactive-list/about#accessibility)
+* [Implementation](/components/cerner-terra-framework-docs/compact-interactive-list/about#implementation)
+* [Properties](/components/cerner-terra-framework-docs/compact-interactive-list/about#properties)
+* [Terra Standards](/components/cerner-terra-framework-docs/compact-interactive-list/about#terra-standards)
+
+## Overview
+
 The `terra-compact-interactive-list` component  provides users a way to render a collection of interactive data in a list format into a single tab stop. The list can be arranged in columns to fit wide space, and the user can navigate between them with the arrow keys.
 
-## Getting Started
+<FixedWidthColumns />
+
+### Features
+Supports:
+* Arranging items in columns.
+* Fixed and responsive width columns.
+* `px`, `em`, and `rem` width units.
+* Vertical and horizontal flow.
+* Minimum and maximum column width for responsive columns set for all columns, or to each column individually.
+* Selectable cells.
+* Interactable elements in cells (one per cell).
+
+Refer to the [examples](/components/cerner-terra-framework-docs/compact-interactive-list/about#examples) section for detailed demonstration of the features supported by the Search Field.
+
+## Design Decisions
+
+The Compact Interactive List:
+  * Supports only one tab stop. When tabbing into the Compact Interactive List, focus will either go to the last visited cell, or the first cell upon initial entry.
+  While on the Compact Interactive List, `Tab` or `Shift+Tab` will exit the list.
+  * Does not maintain consumer data state. The state must be maintained by the consumer.
+  * Uses callbacks to communicate the actions taken by the user.
+  The callbacks are provided by the consumer and therefore the implementation is consumer specific.
+  It is up to the consumer to determine if and what actions should be taken.
+  For example, the user may click on a selectable cell in the Compact Interactive List which calls the _onCellSelect_ callback.
+  For business reasons, the consumer may decide not to honor that cell selection and instead does nothing resulting in the the cell not getting selected in the Worklist Data Grid.
+  * Supports single interactive element per cell.
```

Comment:
Nit:

```suggestion
  * Supports a single interactive element per cell.
```

Reply:
Great suggestion, and much appreciated, but it looks like all wording and styling comment were outdated per this commit by @cldeitrich: https://github.com/cerner/terra-framework/pull/1950/commits/62c72c0098e6edb05ec923b22cf997650ac247bf